### PR TITLE
ansible-lint: Fix name[missing] and name[play] failures

### PR DESCRIPTION
**Pull request by liangwen12year**

### CODE REVIEW

**Comment by liangwen12year** [comment]
This PR is not ready for review.

**Comment by liangwen12year** [comment]
[citest]

File: tests/playbooks/tests_ethtool_features.yml [hunk 1]
```diff
@@ -155,12 +161,17 @@
...
+                  fatal error: configuration error:
+                  connections[0].ethtool.features:
+                  duplicate key 'tx_tcp_segmentation'
+              assert:
                 that:
-                  - '{{  "fatal error: configuration error:
-                      connections[0].ethtool.features: duplicate key
-                      ''tx_tcp_segmentation''" in
-                      __network_connections_result.msg }}'
+                  - errmsg in __network_connections_result.msg
```
Comment (richm) [inline tests/playbooks/tests_ethtool_features.yml:173]: This change is not related to the naming issue

**Comment by richm** [comment]
one minor nit - otherwise - lgtm - but due to the extensive nature of the changes, and the style of the changes, some one from the core network team e.g. @cathay4t @tyll @ffmancera or others should check that the style of the changes is satisfactory and suitable - that is, something like 
```
    - include_tasks: tasks/el_repo_setup.yml
    - name: Include the task 'el_repo_setup.yml'
      include_tasks: tasks/el_repo_setup.yml
```
is fine with me - but I'm not sure if it will be acceptable to the network team

File: tests/playbooks/tests_ethtool_features.yml (hunk 1 above)
Comment (liangwen12year) [reply tests/playbooks/tests_ethtool_features.yml]: I updated the patch, thanks.

**Review by richm — APPROVED** [review]

**Comment by liangwen12year** [comment]
[citest]

**Comment by richm** [comment]
The c8s failures were due to repo network problems.  The c7 and rhel8 tests all passed except for one test that hung, not related.  So imo this is ready to merge.
